spec/controllers/batch_edits_controller_spec.rb

```diff
@@ -29,11 +29,7 @@
       expect(controller).to receive(:add_breadcrumb).with(I18n.t('sufia.dashboard.my.works'), Sufia::Engine.routes.url_helpers.dashboard_works_path)
       get :edit
       expect(response).to be_successful
-      expect(assigns[:terms]).to eq [:creator, :contributor, :description, :keyword, :rights, :publisher,
```

Comment:
Should we leave this assertion in to make sure the batch edit form's terms are what we expect them to be? This would cover my question above re: title, resource_type, and date_created.